master/buildbot/changes/jirapoller.py

```diff
@@ -0,0 +1,100 @@
+from datetime import datetime, timedelta
+
+from twisted.internet import defer
+
+from buildbot.changes import base
+from buildbot.util.logger import Logger
+from buildbot.util.state import StateMixin
+from buildbot.util import httpclientservice
+from buildbot.util import datetime2epoch
+from buildbot.util import ascii2unicode
+
+try:
+    from jira import JIRA
+except ImportError:
+    JIRA = None
+
+log = Logger()
+
+def dt_parse(t):
+    ret = datetime.strptime(t[0:16],'%Y-%m-%dT%H:%M')
+    if t[17]=='+':
```

Comment:
Code formatting in this file is inconsistent. Sometimes there are spaces before and after "=", sometimes there are not etc.